src/todo/helper.lua

```diff
@@ -61,6 +64,10 @@ function todo.show_minimized(player)
     return settings.get_player_settings(player)["todolist-show-minimized"].value
 end
 
+function todo.auto_assign(player)
```

Comment:
You could rename this to "is_auto_assign" or "get_auto_assign" to make it clearer that this function is not actually assigning anything but just reading the settings.